frontend/src/metabase/querying/notebook/components/SummarizeStep/SummarizeStep.tsx

```diff
@@ -22,8 +24,13 @@ export function SummarizeStep({
           {...props}
         />
       </Box>
-      <Box c={color} fw="bold">{t`by`}</Box>
+      {!isMetric && <Box c={color} fw="bold">{t`by`}</Box>}
```

Comment:
With "Default time dimension" left-aligned on mobile, the center-aligned "by" looks weird right now.
Let's style them both in the same way. IMO left-aligned looks better (and all other labels are aligned this way).


![image](https://github.com/user-attachments/assets/98dd0dc2-fb23-4a50-8fee-1ab6280a903b)

Reply:
Fixed